### PR TITLE
Stop context menu disappearing on macOS

### LINKED ISSUE #433: Editor: Context menu not working properly in MacOS + Firefox
The `@HostListener("document:click", ["$event"])` handler is being invoked immediately after the context menu is shown.

### PR DESCRIPTION
**Pull request by msavy**
Attempts to fix #433

### CODE REVIEW

**Comment by msavy** [comment]
Please review, @EricWittmann

**Comment by msavy** [comment]
There's a chance this could be too blunt an instrument, in which case will have to think of another solution. But, I can't /see/ any side effects so far.

File: front-end/studio/src/app/pages/apis/{apiId}/editor/_components/master.component.ts [hunk 1]
```diff
@@ -493,7 +493,12 @@ export class EditorMasterComponent implements OnInit, OnDestroy {
      * menu if it is open.
      */
     @HostListener("document:click", ["$event"])
-    public onDocumentClick(): void {
+    public onDocumentClick(event): void {
```
Comment (EricWittmann) [inline front-end/studio/src/app/pages/apis/{apiId}/editor/_components/master.component.ts]: Can you add the type to the `event` parameter?  Should be `event: MouseEvent`.

**Comment by msavy** [comment]
Your suggestion should be applied now, hopefully.

**Pull request merged**